application/codeaction/handlers.go

```diff
@@ -0,0 +1,77 @@
+package codeaction
+
+import (
+	"context"
+	"sync"
+	"time"
+
+	"github.com/rs/zerolog/log"
+
+	"github.com/snyk/snyk-ls/application/server/lsp"
+)
+
+const debounceDuration = 50 * time.Millisecond
```

Comment:
This should be moved into the handler creation of code action, as it's only relevant there and we don't know if in the future other debounces would use the same value.

Reply:
Done